requirements.txt

```diff
@@ -2,13 +2,15 @@
 appdirs==1.4.3
 certifi==2017.7.27.1
 chardet==3.0.4
+click==6.7
 colorama==0.3.9
 fs==2.0.11
 idna==2.6
 packaging==16.8
 pbr==3.1.1
 psutil==5.3.1
 pyparsing==2.2.0
+python-dotenv==0.7.1
```

Comment:
The requirements are managed by Projectfile / edgy.project, I'll handle this if you prefer (requirements*.txt are rewritten automatically on a regular basis).

Reply:
If I am right in thinking that I just need to add the requirement as another element in this list then I think I can swing it :). If not then maybe I will take you up on that.

![screenshot from 2017-10-15 17-05-51](https://user-images.githubusercontent.com/6644591/31589177-4a57e17a-b1cb-11e7-8fd6-3738061706e0.png)

Reply:
@hartym Am I right about what I have to change? If so I will make the change now.

Reply:
@hartym Actually, I now think as it is a base requirement I would have to add it right after stevedore.

Anyway let me know!

Reply:
Yes, you just have to add it in the "base" requirements, no need to add it also in extras. Usually, I keep the requirements sorted alphabetically, for no good reason other than being a bit maniac. Once it is done, you need either to `pip install edgy.project; edgy-project update` at the root of the project, or I can do it for you if any trouble.

Reply:
It isn't, just did that. We will see soon whether everything is good with the Windows CI now. Also, are the doc changes good?